.ci/scripts/kind-setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
@exekias as soon as we have this in, we should probably disable it in travis?

Reply:
Why not having it in both places?

Reply:
It's a more general comment. If we get the power of the Jenkins Pipeline in our repo, we should disable travis all together so we only have to worry about 1 flaky environment. And I'm sure @kuisathaverat guarantees us 100% availability for Jenkins 🤣

Reply:
Yeah, I guess we can have this discussion once we feel comfortable with jenkins. One point to still having both in the past was that jenkins is not run by default for external contributions.

Reply:
Good point, forgot about this one. Would still be nice to have Jenkins also run for them by default, or at least the once which have signed the CLA.

Reply:
@exekias @ruflin 
One question about the k8s versions we test, Do we need to test versions lower than 1.11? I ask because minikube it is a little tricky to run in a VM, however kind is pretty easy but it only works on k8s +1.11

Reply:
It should be safe to drop versions older than 1.11, cc @odacremolbap

Reply:
If k8s causes trouble, I would suggest to drop it from the initial PR and then follow up.

Reply:
it is working wint k8s versiosn "v1.16.2","v1.15.3","v1.14.6","v1.13.10","v1.12.10", and "v1.11.10"